api.go

```diff
@@ -28,6 +28,11 @@ type responseError struct {
 // easyjson:json
 type errorDetails struct {
 	ErrorCode errorCode `json:"errorCode,omitempty"`
+	FieldViolations []fieldValidation `json:"field_violations"`
```

Comment:
inconsistensy in naming: field`Violations` vs field`Validation`

Reply:
Thx for noticing. Renamed